components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java

```diff
@@ -824,6 +824,7 @@ private static void setMgwAPISecurityAndScopes(ExtendedAPI api, OpenAPI openAPI)
             }
             //generate scopes string
             for (String scope : scopeList) {
+                scope = "\"" + scope + "\"";
```

Comment:
```suggestion
                scope = '"' + scope + '"';
```
Will work here? we can avoid additional string initialization.